src/meshapi/management/commands/replay_join_records.py

```diff
@@ -0,0 +1,74 @@
+from argparse import ArgumentParser
+from dataclasses import asdict, fields
+from datetime import datetime
+from typing import Any
+
+from django.core.management.base import BaseCommand
+from prettytable import PrettyTable
+
+from meshapi.util.join_records import JOIN_RECORD_BASE_NAME, JoinRecord, JoinRecordProcessor
+from meshapi.views.forms import JoinFormRequest, process_join_form
+
+
+class Command(BaseCommand):
+    help = "Replay join form submissions that we may not have accepted properly"
+
+    def add_arguments(self, parser: ArgumentParser) -> None:
```

Comment:
Perhaps for efficiency and damage control reasons it would be good to have a flag to filter submissions by time? (and default it to like -1 week from the current time)

Reply:
Captured in https://github.com/nycmeshnet/meshdb/issues/699